src/Components/Restaurants.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List/index'
+import ListItem from '@material-ui/core/ListItem/index';
+import Restaurant from './Restaurant.js';
+
+
+class Restaurants extends Component {
+    render() {
+        const restaurants = this.props.restaurants;
+        console.log(restaurants);
+        return (
+            <List>
+                {restaurants.map(restaurant =>
+                    <ListItem key={restaurant.id} alignItems='flex-start'>
+                        <Restaurant restaurant={restaurant}/>
```

Comment:
why `Restaurant` itself isn't a `ListItem`? is it reusable enough?

Reply:
should be reusable, just changed the UI to GridList and it still works

Reply:
I changed the list to GridList with GridListItem, so need the robustness